mne/preprocessing/ica.py

```diff
@@ -1643,7 +1652,7 @@ def detect_artifacts(self, raw, start_find=None, stop_find=None,
             the name of function supported by ICA or a custom function.
         ecg_criterion : float | int | list-like | slice
             The indices of the sorted skewness scores. If float, sources with
-            scores smaller than the criterion will be dropped. Else, the scores
+            scores greater than the criterion will be dropped. Else, the scores
```

Comment:
was this an error in the docs or do you change the behavior here?

Reply:
No, here I only changed the doc, since I felt what is actually done in the code (i.e. labelling scores exceeding a threshold) is intuitively what you would expect (in contrast to the above issue, where I found what was promised in the docstring is more intuitive that what was done in the code.)

Reply:
sure, if the docstring had an error - ok.